autoload/go/job.vim

```diff
@@ -1,7 +1,18 @@
-" Spawn returns callbacks to be used with job_start. It is abstracted to be
+let s:cd = exists('*haslocaldir') && haslocaldir() ? 'lcd' : 'cd'
```

Comment:
As I understand it, this doesn't do what you want. The `haslocaldir()` checks if the current window has a local path set already with `:lcd`. So evaluating it once here isn't really that useful. You need to evaluate it when using.

We could add a wrapper function in `util.vim` though.

Reply:
Good catch.